src/styles/blocks/star.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  &__star {
+    width: 16px;
```

Comment:
Consider using an SCSS variable for the width and height values. This will make it easier to update these values across the project if needed.